src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.scss

```diff
@@ -41,47 +41,62 @@ dot-searchable-dropdown {
 }
 
 .searchable-dropdown__data-list {
-    .ui-datalist {
+    .ui-dataview {
         min-height: 321px;
+        padding-bottom: $basic-padding * 4;
```

Comment:
We need to add this padding only if we show pagination, right now is being added at everything:

<img width="411" alt="Screen Shot 2019-07-25 at 10 11 02 AM" src="https://user-images.githubusercontent.com/751424/61890361-8afa6700-aec4-11e9-9e20-18047f680a2e.png">

With the height, I don't want to calculate things or anything, for now just leave the min-height.